data/language/en-GB.txt

```diff
@@ -4568,6 +4568,8 @@ STR_6258    :Clear (transparent)
 STR_6259    :Disabled
 STR_6260    :Show blocked tiles
 STR_6261    :Show wide paths
+STR_6262    :Master volume
+STR_6263    :{SMALLFONT}{BLACK}Toggle all sound on/off
```

Comment:
To me, `toggle` implies a binary state. This is used as a volume slider (scrollbar) tooltip, however. I think the text should reflect this. How about something like "Changes the volume of all sound in the game."?